_sass/custom.scss

```diff
@@ -8,6 +8,12 @@ h1, h2, h3 {
     font-family: Helvetica, Arial, sans-serif;
 }
 
+em {
```

Comment:
I'm finding this hard to read.

Reply:
And confusing that it's the same colour as links.

Reply:
If possible I'd like to address this interactively, as I am a bit close to see what looks good

Reply:
Also noting neither of my reference sites really do this